layouts/experience/single.html

```diff
@@ -24,16 +24,26 @@
             {{ range $xp }} 
 
             <div class="experience {{ if eq .Page.File.UniqueID $currentPageID }} selected {{ end }}" >
-              <a href="{{.Permalink}}">
+              <a href="{{.Permalink | relURL}}">
                 {{/* The context, ".", is now each one of the pages as it goes
                 through the loop */}}
-                <div class="experience__date">{{ .Params.duration }}</div>
-                <div class="experience__title">{{ .Params.jobTitle }}</div>
-                <div class="experience__company">
-                  {{ .Params.company }}.
-                  <span class="experience__location"
-                    >{{ .Params.location }}</span
+                <div class="experience__header">
```

Comment:
here i have copied from experience.html layout, for replicate the same markdown in the single view

Reply:
Nice! Something that will be possible is probably using partials and base layouts to reduce markup duplication - but it's out of the scope for the fix